### PR TITLE
Generic Linux ARM64 package.

### PR DESCRIPTION
**Pull request by dumol**
Scope
=====

Using the Pine64 board again these days, to keep my flat cool. 

But with a Debian-based Armbian this time.

Also as a way to test backup and recovery from scratch.

Changes
=======

Used the SD card with the Ubuntu 16.04 image to build a generic Linux `arm64` package by:

  1) Setting Ubuntu 16.04 as unsupported in `paver.sh` to trigger the generic Linux build.
  2) Adapting the generic `x64` Linux build for `arm64`.

**Drive-by fixes**:
   * Updated `cryptography` to latest version: 2.9.2.
   * Updated `setuptools` to latest version for Python 2.7x: 44.1.1.
   * Minor Alpine adjustments for 3.12 (previous PR was for 3.11).
   * Minor compilation fix for Debian 10 on `arm64`.
   * Travis CI integration for ARM64 builds.  
   * Avoid dangling `bin/bz*` symlinks in the install folder.

How to try and test the changes
===============================

reviewers: @adiroiban 

Please review changes.

Run automated tests, esp. the new Travis CI build for ARM64.

### CODE REVIEW

**Comment by adiroiban** [comment]
SInce this is an open source project, maybe we can try to compile Python for ARM64 on Travis-CI https://docs.travis-ci.com/user/multi-cpu-architectures/

Let me know if you think is a good idea and if you would like to give it a try .

**Comment by dumol** [comment]
> SInce this is an open source project, maybe we can try to compile Python for ARM64 on Travis-CI https://docs.travis-ci.com/user/multi-cpu-architectures/

Thanks, looks interesting! But disappointing to see they are also on Ubuntu 16.04… :-/

**Comment by dumol** [comment]
Something I've worked on in my free time this week… Please take a look!

Should probably rename paver stuff to brink and update PIP as well before merging in master.

**needs-review**

**Comment by adiroiban** [comment]
I will review this after the brink update is merged.
I want to enable travis-ci on ARM64 as part of this PR.

**Comment by dumol** [comment]
> I will review this after the brink update is merged.
> I want to enable travis-ci on ARM64 as part of this PR.

Brink update merged in master and in this branch too.

Looking forward to travis-ci integration too! ;-]

**Comment by dumol** [comment]
Some meaningful changes were lost when merging with master.

**needs-changes**

**Comment by adiroiban** [comment]
we also have setuptools-44.1.1-py2.py3-none-any.whl updated...which is the last version with support for python2

**Comment by dumol** [comment]
As per the updated PR.

**needs-review**

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -23,7 +23,9 @@ jobs:
   ubuntu-20-04:
     # The type of runner that the job will run on
     runs-on: ubuntu-20.04
-    timeout-minutes: 20
+    # Do not run when the following string is used in the commit message.
```
Comment (adiroiban) [inline .github/workflows/main.yaml:26]: this was an experiment in brink... and it does not work for pull requests.

File: brink.sh [hunk 2]
```diff
@@ -865,6 +880,12 @@ if [ "$COMMAND" = "purge" ] ; then
     exit 0
 fi
 
+if [ "$COMMAND" == "detect_os" ]; then
```
Comment (adiroiban) [inline brink.sh:883]: need a comment to describe what this command does... or move it to a function and add comment to that function

File: .github/workflows/main.yaml (hunk 1 above)
Comment (dumol) [reply .github/workflows/main.yaml]: Thanks, have removed those.

File: brink.sh (hunk 2 above)
Comment (dumol) [reply brink.sh:884]: Added a comment. Thanks!

**Pull request merged**